api_docs/changelog.md

```diff
@@ -20,6 +20,13 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 7.0
 
+**Feature level 183**
+
+* [`POST /register`](/api/register-queue), [`GET /events`](/api/get-events):
+  Removed `community_topic_editing_limit_seconds` since time limit for
+  editing topics is controlled by `move_messages_within_stream_limit_seconds`
+  realm setting starting with feature level 162.
```

Comment:
Could we do something similar to what I did in the feature level 167 entry?

>  Removed the realm setting `community_topic_editing_limit_seconds`, which was no longer in use. The time limit for editing topics is controlled by the realm setting `move_messages_within_stream_limit_seconds`, see feature level 162.

Just noting here that, if we don't add a **Changes** note to the `GET /events` documentation, then we don't need to include it here.

Reply:
Updated this for now, but with a slight change since there is no realm setting with name `community_topic_editing_limit_seconds`. It is just that we passed the constant value of 3 days as `realm_community_topic_editing_limit_seconds` in the response.

This does not work well with the `/events` documentation though, but we can update it when we decide what should be the correct change for `/events` part.